packages/jsii-calc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsii-calc",
-  "version": "0.0.0",
+  "version": "3.20.120",
```

Comment:
You could have chosen `3.2.2021` (today's date 🙃) but I guess this is as good as any other pick :)

Reply:
Lol didn't even realize